### ISSUE TITLE
Big Sur

### ISSUE THREAD

**Issue by mattgithub1919** [alexleisyd/lenovo-m90n-hackintosh#3]
你好！感谢分享！

请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。

祝好！

**Comment by alexleisyd** [comment]
I worked on the original config.plist based on the tutorial by Dortania - https://dortania.github.io/OpenCore-Install-Guide/

They have a section for Big Sur. I advise you to follow that tutorial.

**Comment by mattgithub1919** [comment]
Thank you for letting me know. Just want to confirm, config.plist is the only file you modified? I don't need to change the other files? Thank you for your time!

**Comment by mattgithub1919** [comment]
just want to confirm, m90n is Whiskey lake? I guess I can use your config.plist and mix it with the updated files in OC?
![image](https://user-images.githubusercontent.com/30022087/99810648-09dfce00-2b12-11eb-8ecb-0d28154dd5a3.png)

**Comment by alexleisyd** [comment]
That is right. I started off from the base Zip file from them. M90n is actually coffee lake, not whiskey lake but I guess the difference is not dramatic. You need to pay special attention to the graphics part. Good luck!

**Comment by mattgithub1919** [comment]
Hi Alex, 

Thank you for your reply. I managed to boot Big Sur but find that a lot of terminal commands appearing during booting, as shown by the picture below. Could that be some error or it is just expected to happen. Kindly let me know. Thank you!

![IMG_2093](https://user-images.githubusercontent.com/30022087/99919831-d194e700-2ced-11eb-8148-6c7776fa9548.jpg)

**Comment by alexleisyd** [comment]
You can remove -v from the bootarg and it will show the Apple logo instead of the console logs. See [here](https://dortania.github.io/OpenCore-Install-Guide/config.plist/coffee-lake.html#nvram).

**Comment by mattgithub1919** [comment]
Hi Alex, 

Thank you for your info and I have solved the issue. Big Sur is working great now. I find your config.plist is well configured and may I ask a question? 

1. You mentioned ThinkCentre m90n belongs to Intel Desktop Coffee Lake. However, I find your APCI is a bit different from the APCI in the guide for Intel Desktop Coffee Lake. Can you share your insights? (I guess either of them should work as they are almost the same)

<img width="701" alt="image" src="https://user-images.githubusercontent.com/30022087/100120590-1a5fb380-2e46-11eb-96d6-c9c742b692a5.png">

**Comment by alexleisyd** [comment]
The requirement of SSDTs varies from machine to machine. Dortania has a full section just to go through them [here](https://dortania.github.io/Getting-Started-With-ACPI/). I suggest you look at a Youtuber whose explanation is making more sense for us Chinese [here](https://youtu.be/aktkKppp1MA).

**Comment by telus2009** [comment]
> 你好！感谢分享！
> 
> 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> 
> 祝好！

请问安装好了吗？我升级了big sur wifi蓝牙打开不了

**Comment by BrushXue** [comment]
> > 你好！感谢分享！
> > 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> > 祝好！
> 
> 请问安装好了吗？我升级了big sur wifi蓝牙打开不了

我的fork没问题，不过网卡可能跟你用的不一样。等我有时间把0.6.7的更新发上来。

**Comment by eribone** [comment]
> > > 你好！感谢分享！
> > > 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> > > 祝好！
> > 
> > 
> > 请问安装好了吗？我升级了big sur wifi蓝牙打开不了
> 
> 我的fork没问题，不过网卡可能跟你用的不一样。等我有时间把0.6.7的更新发上来。

有0.67的更新吗 最近准备安装这个机器 ，发觉已经0.67了，求配置文件

**Comment by BrushXue** [comment]
> > > > 你好！感谢分享！
> > > > 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> > > > 祝好！
> > > 
> > > 
> > > 请问安装好了吗？我升级了big sur wifi蓝牙打开不了
> > 
> > 
> > 我的fork没问题，不过网卡可能跟你用的不一样。等我有时间把0.6.7的更新发上来。
> 
> 有0.67的更新吗 最近准备安装这个机器 ，发觉已经0.67了，求配置文件

我更新了fork，不过用的网卡和原po的不一样，你得自己对照着改。

**Comment by eribone** [comment]
> > > > > 你好！感谢分享！
> > > > > 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> > > > > 祝好！
> > > > 
> > > > 
> > > > 请问安装好了吗？我升级了big sur wifi蓝牙打开不了
> > > 
> > > 
> > > 我的fork没问题，不过网卡可能跟你用的不一样。等我有时间把0.6.7的更新发上来。
> > 
> > 
> > 有0.67的更新吗 最近准备安装这个机器 ，发觉已经0.67了，求配置文件
> 
> 我更新了fork，不过用的网卡和原po的不一样，你得自己对照着改。

谢谢了，我试试。

**Comment by eribone** [comment]
> > > > > 你好！感谢分享！
> > > > > 请问你的EFI和OpenCore默认的文件有哪里不同？想学习一下如何修改，然后安装Big Sur。
> > > > > 祝好！
> > > > 
> > > > 
> > > > 请问安装好了吗？我升级了big sur wifi蓝牙打开不了
> > > 
> > > 
> > > 我的fork没问题，不过网卡可能跟你用的不一样。等我有时间把0.6.7的更新发上来。
> > 
> > 
> > 有0.67的更新吗 最近准备安装这个机器 ，发觉已经0.67了，求配置文件
> 
> 我更新了fork，不过用的网卡和原po的不一样，你得自己对照着改。

你有没有碰到过，系统闲置时间长了以后，显示器就休眠了，然后再按键盘，显示器也无法点亮。有时候需要插拔typec的显示线重连的方式，才能重新点亮显示器。有没有什么修正方式。

**Comment by BrushXue** [comment]
我连的DP线，没有遇到过这种情况。
